### PR TITLE
Feature/decrypt messages

### PR DESCRIPTION
**Pull request by josetheis-stackit**

### CODE REVIEW

File: src/features/channels/components/channel-list/channel-list-item/ChannelListItem.tsx [hunk 1]
```diff
@@ -31,16 +33,42 @@ const ChannelListItem = ({
...
+          message: {
+            ...message,
+            decryptedMessage: await encryptionService.decryptMessage(
+              message.message,
+              passphrase,
+              privateKeyArmored
+            ),
+            decryptedReplyMessage:
+              message.replyMessage &&
+              (await encryptionService.decryptMessage(
```
Comment (mmiranda-stackit) [inline src/features/channels/components/channel-list/channel-list-item/ChannelListItem.tsx:64]: This is very hard to understand, I'd suggest to move most of this logic to actions, or put it into a variable and reference it from here

File: src/features/channels/components/chat-content/ChatContent.tsx [hunk 2]
```diff
@@ -56,28 +49,24 @@ const ChatContent = () => {
...
-            {replyMessage && replyRecipientAlias && (
-              <MessageReply name={replyRecipientAlias} message={replyMessage} color="#A36300" />
-            )}
-          </Message>
-        )
-      )}
+      {sortedMessages.map((message, index) => (
+        <Message
+          // TODO: the backend is not giving us any way to differentiate between messages... Too bad!
+          // eslint-disable-next-line react/no-array-index-key
```
Comment (mmiranda-stackit) [inline src/features/channels/components/chat-content/ChatContent.tsx:55]: Do not use inline eslint disable commands. It is better to follow the rule, or there is a very good reason to not to do so, create a general rule to disable it

File: src/features/channels/components/chat-content/ChatContent.tsx [hunk 3]
```diff
@@ -56,28 +49,24 @@ const ChatContent = () => {
...
+          key={index}
+          message={message}
+          color="#A36300"
+        >
+          {/* TODO: there must be a better way to achieve this */}
```
Comment (mmiranda-stackit) [inline src/features/channels/components/chat-content/ChatContent.tsx:60]: If we think that we have some tech debt to resolve, it is a good idea to create tasks and put the id to reference the pending work.

File: src/features/channels/components/chat-content/ChatContent.tsx [hunk 4]
```diff
@@ -56,28 +49,24 @@ const ChatContent = () => {
...
+          {message.decryptedReplyMessage && message.replyRecipientAlias && (
+            <MessageReply
+              name={message.replyRecipientAlias}
+              message={message.decryptedReplyMessage}
+              color="#A36300"
```
Comment (mmiranda-stackit) [inline src/features/channels/components/chat-content/ChatContent.tsx:65]: Are we going to make colors dynamic? if so, we should not hardcode hexa codes here.

**Review by mmiranda-stackit — DISMISSED** [review]
Approved with some comments

File: src/features/channels/components/chat-content/ChatContent.tsx (hunk 2 above)
Comment (josetheis-stackit) [reply src/features/channels/components/chat-content/ChatContent.tsx:55]: There's no messageId when a new message gets added to a channel through the socket... So, for this time I had to disable eslint for that line. This is a design problem from the backend

**Pull request merged**